src/index.ts

```diff
@@ -2,32 +2,33 @@ import { Dispatch, Middleware, MiddlewareAPI } from "redux";
 import {
   ActionCreatorWithPreparedPayload, createAction, PayloadAction, PayloadActionCreator, PrepareAction,
 } from "@reduxjs/toolkit";
-import { isFunction, merge } from "lodash";
+import { merge } from "lodash";
 import { call, CallEffect, SagaReturnType } from "redux-saga/effects";
 import { ActionCreatorWithPayload, _ActionCreatorWithPreparedPayload } from "@reduxjs/toolkit/dist/createAction";
 import { ArgumentError } from "./ArgumentError";
 import { ConfigurationError } from "./ConfigurationError";
 
+const promiseActionSymbol = Symbol("redux-saga-promise-action");
```

Comment:
It is mandatory to export it to make it globally available, so we allow others still to mimick our action structure.

Reply:
@teneko fixed